qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -42,11 +21,12 @@ def __init__(self, name, instrument, start, stop, npts, channel):
         self._channel = channel
         self.names = ('magnitude',
                       'phase')
-        self.labels = ('{} magnitude'.format(instrument._vna_parameter),
-                       '{} phase'.format(instrument._vna_parameter))
+        self.labels = ('{} magnitude'.format(instrument.short_name),
```

Comment:
Can you remind me of name, full_name vs short name

Reply:
full name is 'Instrument name'_'Channel name' and short_name is 'Channel name'.
In my case full name is VNA_S21 for and short_name is S21.

Reply:
Ok thats cool